code/game/objects/structures/watercloset.dm

```diff
@@ -488,6 +488,22 @@
 		return
 	return ..()
 
+/obj/structure/sinkframe/wrench_act_secondary(mob/living/user, obj/item/tool)
+	. = ..()
+	tool.play_tool_sound(src)
+	deconstruct()
+	return TRUE
+
+/obj/structure/sinkframe/deconstruct()
+	if(!(flags_1 & NODECONSTRUCT_1))
+		drop_materials()
+	return ..()
+
+/obj/structure/sinkframe/proc/drop_materials()
+	for(var/datum/material/material as anything in custom_materials)
+		new material.sheet_type(loc, FLOOR(custom_materials[material] / MINERAL_MATERIAL_AMOUNT, 1))
+	return
```

Comment:
```suggestion

```
No need for empty returns at the end of the proc. They're implied and don't bring up any new information.